lib/LiquidTemplate.class.php

```diff
@@ -131,6 +136,17 @@ public static function getTags()
     }
 
 
+    /**
+     *
+     *
+     * @return array
+     */
+    public static function getContext()
+    {
+        return self::$_context;
+    }
```

Comment:
I wonder if adding this static function can be avoided. We already have more than enough static variables.